### PR TITLE
Small clarifications in the docs

### PR DESCRIPTION
**Pull request by eallrich**
While following the tutorial and quick start documentation I noticed a couple possible improvements. Hopefully these contributions are helpful. Thank you for the SDK!

### CODE REVIEW

File: doc/source/quick_start.rst [hunk 1]
```diff
@@ -282,8 +282,6 @@ Get file metadata
 Copy file
 =========
 
-Please switch to  :meth:`b2sdk.v2.Bucket.copy`.
```
Comment (eallrich) [inline doc/source/quick_start.rst:285]: I couldn't import from `b2sdk.v2` and I didn't see any other references to a `v2` of the API so I've pulled this line out. Might this be for an upcoming release instead?

Comment (ppolewicz) [reply doc/source/quick_start.rst:285]: it should be `b2sdk.v1.Bucket.copy` - v2 is not released yet and we don't need it any time soon

Comment (eallrich) [reply doc/source/quick_start.rst:285]: Thank you. I've restored the reference and switched it to `v1`.

**Comment by ppolewicz** [comment]
All right, thanks for your contribution, documentation always needs some love!

**Pull request merged**